src/steps/writing/creation/dotVSCode.ts

```diff
@@ -50,5 +60,17 @@ export async function createDotVSCode(options: Options) {
 			"eslint.rules.customizations": [{ rule: "*", severity: "warn" }],
 			"typescript.tsdk": "node_modules/typescript/lib",
 		}),
+		"tasks.json": await formatJson({
+			tasks: [
+				{
+					detail: "Build the project",
+					label: "build",
+					problemMatcher: [],
+					script: "build",
```

Comment:
[Bug] Relying `build` -> `pnpm run build` means it'll wait for type checking per `tsup.config.ts`. That's unnecessary IMO - oftentimes I'll have compile errors while working on an in-progress change. If a pre-launch task is to exist, IMO it should be as fast as possible. So this should `--dts false`.

Reply:
I was just trying to test this change out by running the initialize command and testing it locally; I'm getting this error:

```
pnpm i
pnpm run build
pnpm run initialize


> create-typescript-app@1.52.1 initialize /home/john/code/github/create-typescript-app
> tsx ./src/bin/index.js --mode initialize


node:internal/process/esm_loader:40
      internalBinding('errors').triggerUncaughtException(
                                ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/john/code/github/create-typescript-app/src/bin/index.js' imported from /home/john/code/github/create-typescript-app/
    at __node_internal_captureLargerStackTrace (node:internal/errors:497:5)
```

I'm pretty sure it didn't do that last time I tested - is CTA working okay?  Or am I misremembering how to test?

Reply:
Oh! I think CTA is broken. Wow. It should be trying to `tsc ./src/bin/index.ts`.

Reply:
Oh no!

Reply:
#1168 -> #1170